Source/Public/WorldHandler.h

```diff
@@ -0,0 +1,73 @@
+﻿#pragma once
+
+#include <Engine/World.h>
+#include <WorldPartition/WorldPartition.h>
+
+class FWorldHandler
```

Comment:
since it's only used in the CPP file of the commandlet, you can move this at the top of the CPP file (below the includes)

Reply:
done.